Sources/DonationViewController.swift

```diff
@@ -0,0 +1,225 @@
+/*****************************************************************************
+ * DonationViewController.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2013 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Felix Paul Kühne <fkuehne # videolan.org>
+ *          Gleb Pinigin <gpinigin # gmail.com>
```

Comment:
Pretty sure Gleb and me never contributed something in swift ;)

Reply:
nicely spotted